pkg/plugins/scms/github/pullrequest.go

```diff
@@ -47,7 +47,7 @@ const PULLREQUESTBODY = `
 
 ## Remark
 
-This pull request was automatically created using [Updatecli](https://www.updatecli.io).
+This pull request was automatically created from [{{ .Filename }}]({{ .FileURL }}) using [Updatecli](https://www.updatecli.io).
```

Comment:
```suggestion
This pull request was automatically created from the manifest [{{ .Filename }}]({{ .FileURL }}) using [Updatecli](https://www.updatecli.io).
```

Reply:
Also in the case of autodiscovery, there is no file manifest so it's better to have a condition such as if Filename has value set